src/Agent.Plugins/TFCliManager.cs

```diff
@@ -275,7 +275,17 @@ public async Task WorkspaceDeleteAsync(ITfsVCWorkspace workspace)
 
         public async Task WorkspaceNewAsync()
         {
-            await RunCommandAsync("vc", "workspace", "/new", "/location:local", "/permission:Public", WorkspaceName);
+            var useServerWorkspace = StringUtil.ConvertToBoolean(ExecutionContext.Variables.GetValueOrDefault("build.useserverworkspaces")?.Value ?? "false");
```

Comment:
`GetValueOrDefault`, does that have a default?  If so why would we need the '?? "false"'.

Reply:
nit: I would probably make it a `bool useServerWorkspace`